src/content/docs/enterprise/translation-process/offline-translation.mdx

```diff
@@ -111,48 +111,48 @@ To upload a file with translations, follow these steps:
 </Steps>
 
 ### Uploading XLIFF Translations
 
-Once you've finished translating an XLIFF file earlier exported from your project, you can upload completed translations back to Crowdin Enterprise.
-
-XLIFF file with translations can be uploaded the following way:
+Once you've finished translating an XLIFF file exported from the project, you can upload the completed translations back to Crowdin Enterprise using the following steps:
 
 <Steps>
   1. Open your project and select a language.
-  2. Click <Icon name="mdi:dots-vertical" class="inline-icon"/> in the upper-right corner.
+  2. Click <Icon name="mdi:dots-vertical" class="inline-icon" /> in the upper-right corner.
   3. Select **Upload XLIFF translations**. <Image src={downloadUploadFilesXliff} alt="Upload Translations in XLIFF" />
-  4. Configure the [Advanced Import Settings](#advanced-import-settings) toward the uploaded translations.
-  5. Click **Select File**.
-  6. Select an XLIFF file with translations on your machine and click **Open**.
+  4. Adjust the [Advanced Import Settings](#advanced-import-settings) as needed.
+  5. Click **Select File** and choose the XLIFF file with translations from your device.
 </Steps>
 
 ### Advanced Import Settings
 
-By default, translations that match the original strings and translations identical to the existing ones are skipped during the upload. Configure the system behavior by selecting suitable options in the appeared dialog.
+When uploading translations, you can adjust the following import options to control how the system handles the new content:
+
+* **Allow target translation to match source** – Enable this if the translation is intentionally the same as the source (e.g., for product names or code terms).
+* **Approve added translations** – Automatically approve the uploaded translations if you're confident in their accuracy.
+* **Translate hidden strings** – Include strings that are hidden from the Editor but still require translation (e.g., for system use or placeholders).
 
 <Aside title="Permissions">
-  Some of the import options shown below may be available only to project members with manager permissions (or higher).
+  Some of the import options may be available only to project members with manager permissions (or higher).
 </Aside>
 
 ## Common Questions
 
-**My XLIFF file was translated offline. Now It contains finished translations, but when I try to upload it back to Crowdin Enterprise, the translations are not displayed in the project. What might be the reason?**
-
-According to the official documentation, `<target>` elements in the XLIFF file contain the `state` attribute that indicates the current translation state for each string.
+<QuestionAnswer title="My XLIFF file was translated offline. Now It contains finished translations, but when I try to upload it back to Crowdin Enterprise, the translations are not displayed in the project. What might be the reason?">
```

Comment:
Let's move this `QuestionAnswer` block to the "Uploading XLIFF Translations" section